examples/examples.sh

```diff
@@ -0,0 +1,26 @@
+for folder in */; do
```

Comment:
this simple script generates the Cargo.toml file for each example to make it easy to add new examples or modify the template.  we will probably re-export legion-prefab and prefab-format from amethyst in the future but there are macros that have paths to them now and code would get messy to include re-exports in the examples.

this dependencies list is a union of all examples right now.  users can audit it themselves or we could add a `cargo udeps` run to the end.  I think it's alright to have some examples with excess dependencies for now.